src/index.js

```diff
@@ -82,6 +82,10 @@ export default class Voices {
     return this.byLangCode('pl-PL')
   }
 
+  french () {
+    return this.byLangCode('fr-FR')
+  }
+
```

Comment:
you should use the method `byLang` instead of `byLangCode` because we have Canadian French as well. Take a look at the `english` method.